### ISSUE TITLE
UTM at a campaign level

### ISSUE THREAD

**Issue by Ivshti** [heyAuraDev/adex-platform#472]
**Is your feature request related to a problem? Please describe.**
After #468 and AIP31, targeting is now at a campaign level. Which makes ad unit-specific UTMs pretty useless, which is why UTMs will be at a campaign level too after https://github.com/AdExNetwork/adex-adview-manager/issues/32

**Describe the solution you'd like**
- [ ] remove auto UTM from the ad unit creation UI
- [ ] checkbox in the campaign creation UI to add UTMs, on by default with a help icon that points to a zendesk tutorial that shows how UTMs are formed

**Comment by Ivshti** [comment]
closing in favor of https://github.com/AdExNetwork/adex-platform/issues/476 - it's a more detailed description of this, but not assuming that this will be built into the AdView because it won't be (it also explains why)
